README.adoc

```diff
@@ -188,7 +188,7 @@ class the Initializr creates (at
 `src/main/java/com/example/consumingrest/ConsumingRestApplication.java`):
 
 ====
-[src,Java]
+[source,java,tabsize=2]
```

Comment:
The lower case is correct. We prefer not to replace tabs with spaces for accessibility reasons.

Reply:
Basically I wanted to fix syntax highlighting which looks like this right now: 

![image](https://user-images.githubusercontent.com/16988850/72989544-bff16300-3dee-11ea-80ba-2b8acbe349cb.png)

I found that `[source,java,tabsize=2]`  from `README.adoc` is working, but if you want I can delete the **tabsize** option. 

Are you going to be happy with `[source,java]`?

Reply:
> Are you going to be happy with [source,java]?

Yes, that's what I meant. The highlighting is broken by the "J" not the tabsize.